Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -476,7 +476,7 @@ class MediaControlTests: QuickSpec {
                 context("when kMediaControlPluginsOrder is passed") {
                     it("renders the plugins following the kMediaControlPluginsOrder order") {
                         let pluginName = "FullscreenButton"
-                        core.options[kMediaControlPluginsOrder] = pluginName
+                        core.options[kMediaControlPluginsOrder] = [pluginName]
```

Comment:
Do you think we should add more tests, for cases where we have more than 1 plugin name being passed as option? Or do you think this one is enough?

Reply:
Hey @jademcosta, we just did it.